cvat/apps/engine/tests/utils.py

```diff
@@ -128,24 +128,49 @@ def setUp(self):
         super().setUp()
         self.client = self.client_class()
 
-    def _get_request(self, path: str, user: str, *, data: dict[str, Any] | None = None) -> Response:
+    def _get_request(self, path: str, user: str, *, query_params: dict[str, Any] | None = None) -> Response:
         with ForceLogin(user, self.client):
-            response = self.client.get(path, data=data)
+            response = self.client.get(path, data=query_params)
         return response
 
     def _delete_request(self, path: str, user: str):
         with ForceLogin(user, self.client):
             response = self.client.delete(path)
         return response
 
-    def _post_request(self, path: str, user: str, *, data: dict[str, Any] | None = None):
+    def _post_request(
+        self,
+        path: str,
+        user: str,
+        *,
+        format: str = "json",  # pylint: disable=redefined-builtin
+        query_params: dict[str, Any] = None,
+        data: dict[str, Any] | None = None
+    ):
+        if query_params:
+            # Note: once we upgrade to Django 5.1+, this should be changed to pass query_params
```

Comment:
I guess we need to raise an exception here if the django version is 5.1+, because otherwise this comment will be forgotten (since [changes](https://github.com/django/django/blob/5773bc9cf929f4648da30cb9c472b816bb4db771/django/test/client.py#L674) are backward compatible).

Reply:
I don't really want to force this change to be done _immediately_ upon upgrading Django. Upgrading can be complicated, so I'd rather not pile more work on whoever's doing it.

The note is just a reminder in case anyone touches this in the future. If it's forgotten, it's not a big deal. The `query_params` emulation is not broken, it's just redundant.